settings/ui/tests/run-acceptance-test.sh

```diff
@@ -40,8 +40,10 @@ clean_up() {
 
 trap clean_up SIGHUP SIGINT SIGTERM EXIT
 
-cp -r "$WEB_PATH"/tests/acceptance/stepDefinitions "$testFolder"
-cp "$WEB_PATH"/tests/acceptance/setup.js "$testFolder"
+cp -r "$WEB_PATH"/tests/acceptance/* "$testFolder"
+rm -r $testFolder/node_modules
```

Comment:
Could we do `cp -r !(node_modules) "$WEB_PATH"/tests/acceptance/*` or the likes instead of copying and removing those gazillion files? :)

Reply:
yes, and also I think we don't need one `yarn install` in drone